client/src/main/python/slipstream/wrappers/CloudWrapper.py

```diff
@@ -77,16 +77,13 @@ def start_node_instances(self):
                                           util.toTimeInIso8601(time.time()))
             return
 
-        cloud_service_name = self._get_cloud_service_name()
-        user_info = self._get_user_info(cloud_service_name)
-
-        provisioning_stop_time = self._get_provisioning_timeout_time(user_info)
-
-        self._start_nodes_and_clients(user_info, nodes_instances)
+        self._start_nodes_and_clients(
+            self._get_user_info(self._get_cloud_service_name()), nodes_instances)
 
         self.clean_local_cache()
+
         nodes_instances_list = self._get_node_instances_to_follow(nodes_instances.keys())
-        self._check_provisioning(nodes_instances_list, provisioning_stop_time)
+        self._check_provisioning(nodes_instances_list, self._get_provisioning_timeout_time())
```

Comment:
OK. I recall why I did that.  I wanted to make `_get_provisioning_timeout_time()` just a getter of the precomputed timeout as a wall-clock time... but finally didn't do that :(  I'm reverting back with the use of the method-local variable computed before `self. _start_nodes_and_clients()`